pyproject.toml

```diff
@@ -1,76 +1,121 @@
+[build-system]
+build-backend = "poetry.core.masonry.api"
+requires = ["poetry-core>=1.1.0"]
+
+[tool.coverage.report]
+show_missing = true
+
+[tool.coverage.run]
+source = ["esak"]
+
 [tool.poetry]
-name = "esak"
-version = "1.3.2"
-description = "Python wrapper for the Marvel API."
 authors = ["Brian Pepple <bdpepple@gmail.com>"]
-license = "MIT"
-maintainers = ["Brian Pepple <bdpepple@gmail.com>"]
-readme = "README.rst"
-packages = [
-	{ include = "esak" },
+classifiers = [
+  "Framework :: Pydantic :: 2",
+  "Intended Audience :: Developers",
+  "License :: OSI Approved :: MIT License",
+  "Natural Language :: English",
+  "Operating System :: OS Independent",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12",
+  "Programming Language :: Python",
+  "Topic :: Utilities",
+  "Typing :: Typed"
 ]
+description = "Python wrapper for the Marvel API."
 include = [
-    { path = "tests", format = "sdist" }
+  {format = "sdist", path = "tests"}
 ]
-classifiers=[
-    "Intended Audience :: Developers",
-    "Natural Language :: English",
-    "Operating System :: OS Independent",
-    "Programming Language :: Python :: 3.10",
-    "Programming Language :: Python :: 3.11",
-    "Programming Language :: Python :: 3.12",
-    "Programming Language :: Python",
-    "Topic :: Utilities",
+license = "MIT"
+maintainers = ["Brian Pepple <bdpepple@gmail.com>"]
+name = "esak"
+packages = [
+  {include = "esak"}
 ]
+readme = "README.md"
+version = "1.3.2"
 
 [tool.poetry.dependencies]
-python = "^3.10"
 pydantic = "^2.8.0"
+python = "^3.10"
 requests = "^2.26.0"
 
 [tool.poetry.group.dev.dependencies]
-pytest-cov = "^2.12.1"
-black = "^21.8b0"
-requests-mock = "^1.9.3"
-flake8 = "^3.9.2"
-tox = "^3.24.3"
-pre-commit = "^2.14.1"
-isort = "^5.9.3"
-seed-isort-config = "^2.2.0"
-pyupgrade = "^2.29.0"
-flake8-builtins = "^1.5.3"
-flake8-docstrings = "^1.6.0"
-flake8-rst-docstrings = "^0.2.3"
-
-[tool.poetry.group.docs.dependencies]
-sphinx-rtd-theme = "^1.3.0"
-sphinxcontrib-napoleon = "^0.7"
+pre-commit = "^3.8.0"
```

Comment:
Seem to be missing the deps needed for linting (like ruff), which should be in a linting group or dev group.

Reply:
Added ruff to dev group with pre-commit